src/ui/pages/home.page.ts

```diff
@@ -0,0 +1,32 @@
+import { PortalPage } from './portal.page.js';
+
+class HomePage extends PortalPage {
+  //   uniqueElement = '//strong[.="AQA User"]';
+  uniqueElement = 'strong';
+
+  readonly 'Orders button' = "//button[@id='orders-from-home']";
+  readonly 'Products button' = "//button[@id='products-from-home']";
+  readonly 'Customers button' = "//button[@id='customers-from-home']";
+
+  async openOrders() {
+    await $(this['Orders button']).click();
+  }
+
+  async openProducts() {
+    await $(this['Products button']).click();
+  }
+
+  async openCustomers() {
+    (await $(this['Customers button'])).click();
+  }
+
+  async clickOnViewDetailsbutton(moduleName: 'Products' | 'Customers' | 'Orders') {
+    await this.click(this[`${moduleName} button`]);
+  }
+
+  async verifyUser() {
+    expect(this.uniqueElement).toHaveText('AQA User');
+  }
```

Comment:
В сервис вьіносим проверки
Делаем метод динамичньім для работьі с различньіми юзерами
```suggestion
  async verifyUser(username: string) {
    expect(this.uniqueElement).toHaveText(username);
  }
```